src/components/FilterBarPopup/FilterBarPopup.mdx

```diff
@@ -0,0 +1,395 @@
+- {<Link href="#usefilterbarpopup-hook" target="_self">useFilterBarPopup hook</Link>}
+- {<Link href="#changelog" target="_self">Changelog</Link>}
+
+## Overview
+
+<Description of={FilterBarPopupStories} />
+
+`FilterBarPopup` is a wrapper component for filter components and individual form
+input components. The DS considers the following components as filter components:
+`CheckboxGroup`, `MultiSelectGroup`,`RadioGroup`, `Checkbox`, `DatePicker`,
+`MultiSelect`, `Radio`, `Select`, `Slider`, `TextInput`, and `Toggle`.
+
+The wrapped components/component groups can be displayed in a column or in a row
+layout.
+
+It is recommended to use real-time filtering in conjunction with the `totalResults`
+prop to update the `Show X results` button with X being the total number of results
+associated to the results list.
+
+If the consuming app is not using the above, then `FilterBarPopup` will render the
+`Show results` button and can render an _optional_ `Clear all filters`
+button which is controlled by the `onClear` prop. The `Show results` button will
+call the `onSubmit` function if passed and close the modal.
+
+Related component: [FilterBarInline](../?path=/docs/components-form-elements-FilterBarInline--docs)
+
+## Component Props
+
+<Canvas of={FilterBarPopupStories.WithControls} />
+
+<ArgTypes of={FilterBarPopupStories.WithControls} />
+
+## Accessibility
+
+The `FilterBarPopup` component is a wrapper component for components with filter
+functionality. The accessibility of `FilterBarPopup` depends on the passed child
+components accessibility. See the accessibility details of valid FilterBarPopup child
+components:
+
+- [CheckboxGroup](../?path=/docs/components-form-elements-checkboxgroup--docs#accessibility)
+- [MultiSelectGroup](../?path=/docs/components-form-elements-multiselectgroup--docs#accessibility)
+- [RadioGroup](../?path=/docs/components-form-elements-radiogroup--docs#accessibility)
+- [Checkbox](../?path=/docs/components-form-elements-checkbox--docs#accessibility)
+- [DatePicker](../?path=/docs/components-form-elements-datepicker--docs#accessibility)
+- [MultiSelect](../?path=/docs/components-form-elements-multiselect--docs#accessibility)
+- [Radio](../?path=/docs/components-form-elements-radio--docs#accessibility)
+- [Select](../?path=/docs/components-form-elements-select--docs#accessibility)
+- [Slider](../?path=/docs/components-form-elements-slider--docs#accessibility)
+- [TextInput](../?path=/docs/components-form-elements-textinput--docs#accessibility)
+- [Toggle](../?path=/docs/components-form-elements-toggle--docs#accessibility)
+
+## FilterBarPopup Heading
+
+The `FilterBarPopup` component can be rendered with and without a heading. By
+default, when a string is passed to the `heading` prop, the
+`FilterBarPopup` component will render an `h2` element sized as a `heading5`
+heading. If this causes accessibility issues in your app, you can set a custom
+heading by passing in a DS `Heading` component.
```

Comment:
Change in functionality.  The `heading` will still be optional, but we are going to give it a default valiue so that a heading is always rendered in the modal.  Change this paragraph to the following:
```
The `FilterBarPopup` component will always render a heading using the default 
"Filter criteria" label value. The default label value can be overridden using the 
`heading` prop. By default, when a string is passed to the `heading` prop, the 
`FilterBarPopup` component will render an `h2` element sized as an `heading5` 
heading. If this causes accessibility issues in your app, you can set a custom 
heading by passing in a DS `Heading` component.